includes/Reconciliation/ReconciliationServiceItem.php

```diff
@@ -45,4 +47,21 @@ public function getRevision() {
 		return $this->revision;
 	}
 
+	/**
+	 * Replace the item and revision ID with those of the given entity revision.
+	 *
+	 * This should be used after the caller has saved the reconciliation service item.
+	 * The entity revision is assumed to be based on the previous {@link #getItem item};
+	 * in particular, the caller must ensure that the entity is an Item,
+	 * and that it has the same URL statement
+	 * (i.e. it is correct for a {@link ReconciliationService} to still return
+	 * this {@link ReconciliationServiceItem} instance for the same URL).
+	 */
+	public function replaceWith( EntityRevision $entityRevision ) {
```

Comment:
Sounds okay. Should we go for `updateFromRevision`, or something similar? ^^ (I initially added just `setRevision( int $revisionId )`, but then realized the Item needed to be set as well, and at that point “replace” made sense to me, since both members of the object are being replaced.)

Reply:
updateFromRevision / setRevision both works for me :+1:

Reply:
Hmm, good question.. `finalize`? but might give the wrong ideas depending on who reads it. :D 

Can't really come up with anything better i think :)

Reply:
I went with `updateFromEntityRevision()` so it doesn’t look so similar to `getRevision()` (maybe `getRevision()` should be `getRevisionId()` but I don’t want to do that here).